warehouse/models/mart/transit_database/dim_transit_data_quality_issue_types.sql

```diff
@@ -0,0 +1,21 @@
+{{ config(materialized='table') }}
+
+WITH int_transit_database__issue_types AS (
+    SELECT * FROM {{ ref('int_transit_database__issue_types') }}
+),
+
+dim_transit_data_quality_issue_types AS (
+    SELECT
+        key,
+        source_record_id,
+        dataset_type,
+        transit_data_quality_issue,
```

Comment:
This looks like it's also a foreign key -- this is the `source_record_id` for an issue. In this case, it seems like this types table is literally just a table that stores a string value (`name`) so what I would probably do here is just stick this on the other table as an issue type. Per the below query it looks like this relationship is always 1:1 (they pick just one type for each issue), so you can probably just do this lookup and not have this as a dedicated table in the mart. (If we did want a mart table, we would want to drop this foreign key and dedupe to just the type names.... I think that this is basically just a way to have a single select field in Airtable though so there's no need for us to maintain this normalization in the mart.)

```
SELECT transit_data_quality_issue, COUNT(source_record_id) FROM `cal-itp-data-infra-staging.soren_mart_transit_database.dim_transit_data_quality_issue_types`
GROUP BY 1 
ORDER BY 2 DESC
```

Reply:
Resolved in [b9b4c3f](https://github.com/cal-itp/data-infra/pull/2781/commits/b9b4c3fbe32bc8b0c4d65d4417a209a017ae62f3)